book/types_of_data.md

```diff
@@ -395,11 +396,30 @@ To remove rows from a table, you'll commonly use the `select` command with row n
 
 There are numerous other commands for selecting and reducing the data in tables, records and lists.
 
+## Closures
+
+Closures are anonymous functions that can close over variables outside their scope and can be passed as a value.
```

Comment:
Maybe we could lighten the CS lingo for the first sentence

```suggestion
Closures are anonymous functions that can be passed a value through paramters and *close over* (i.e. use) a variable outside its scope.
```

Reply:
There is a small typo: paramters -> parameters :)

Reply:
Also made a grammar mistake with `its` instead of `theirs`. Peak Verschlimmbessern

Reply:
I have to say, reading through the sentence, I still like the original a bit more.
That functions can "be passed a value through parameters" is not astounding, but that they "can be passed as a value" is worth mentioning.

Reply:
Done